Dockerfile

```diff
@@ -62,12 +67,13 @@ RUN apt-get update && \
     rm -rf /var/lib/apt/lists/*
 
 COPY --from=backend-build-stage /tmp/dist /opt/rotki
+COPY --from=colibri-build-stage /tmp/dist/colibri/release/colibri /opt/rotki
 COPY --from=frontend-build-stage /app/app/dist /opt/rotki/frontend
 
 RUN APP=$(find "/opt/rotki" -name "rotki-core-*-linux"  | head -n 1) && \
     echo "${APP}" && \
     ln -s "${APP}" /usr/sbin/rotki && \
-    ln -s /opt/rotki/colibri/release/colibri /usr/sbin/colibri
+    ln -s /opt/rotki/colibri /usr/sbin/colibri
```

Comment:
this is what was copying more stuff?

Reply:
This part is just linking the binary to `/usr/sbin` the problem was with the copy command that was copying the whole colibri release directory from the build stage to the final image stage.